app/models/previous_id.rb

```diff
@@ -1,7 +1,7 @@
 class PreviousId < Base
   attribute :prev_id, :string
 
-  validates :prev_id, presence: { message: "Please enter your previous teacher ID number" } # is this numeric?
+  #validates :prev_id, presence: { message: "Please enter your previous teacher ID number" } # is this numeric?
```

Comment:
Can we just delete this?

Reply:
I dont have any validation parameters for this input at the moment, might be numerical, might be alpha, although it is optional, it might be used against the CRM, so my thinking is we should do some type of validation once we know what the field looks like, so if they do enter input it must be correct, left this as a reminder.